README.md

```diff
@@ -18,9 +18,11 @@ The repository is hosted in [GitHub](https://github.com/martincostello/website):
 
 This project is licensed under the [Apache 2.0](https://github.com/martincostello/website/blob/main/LICENSE) license.
 
-## Building and Testing
+## Building and Testing (Windows)
 
-To build and test the website run the following command:
+Download the source code (by clicking 'clone' and 'Download zip' above)
+Extract the downloaded file and navigate into the extracted folder.
+Open Powershell there and enter the following commands to build and test the website:
 
 ```powershell
 ./build.ps1
```

Comment:
Using Git to clone the site is easier than explaining how to use the GitHub UI to download and unpack a ZIP.

```suggestion
To build and test the website run the following commands from your terminal of choice:

```powershell
git clone https://github.com/martincostello/website.git
cd website
./build.ps1
```